mozdownload/cli.py

```diff
@@ -45,60 +38,41 @@ def cli():
                         default=scraper.BASE_URL,
                         metavar='BASE_URL',
                         help='The base url to be used, default: "%(default)s"')
+    parser.add_argument('--build-number',
+                        dest='build_number',
+                        type=int,
+                        metavar='BUILD_NUMBER',
+                        help='Number of the build (for candidate, daily, and tinderbox builds)')
```

Comment:
Those changes here are only for the correct order, so it applies to the order of arguments for the FactoryScraper.